### PR TITLE
Proper changes history

### PR DESCRIPTION
**Pull request by carllopez**

### CODE REVIEW

File: crc/models/study.py [hunk 1]
```diff
@@ -165,18 +186,14 @@ def update_model(self, study_model: StudyModel):
...
-
-        # if study_model.changes_history:
-        #     changes_history = json.loads(study_model.changes_history)
-        #     changes_history.append(change)
-        # else:
-        #     changes_history = [change]
-        # study_model.changes_history = json.dumps(changes_history)
+        study_event = StudyEvent(
+            study=study_model,
+            status=status,
```
Comment (ajlouie) [inline crc/models/study.py:191]: The study status is being saved here in the study event, but doesn't the status of the study itself need to change as well?

File: crc/models/study.py [hunk 2]
```diff
@@ -165,18 +186,14 @@ def update_model(self, study_model: StudyModel):
         if status == StudyStatus.open_for_enrollment:
             study_model.enrollment_date = self.enrollment_date
```
Comment (ajlouie) [inline crc/models/study.py:187]: Should changes to enrollment date be stored in study events as well?

Comment (carllopez) [reply crc/models/study.py:187]: It is tracked with a timestamp, but not with the user entered date. As this change **most** of the time should be final, I opted for having the enrollment date reside in the study itself.

File: crc/models/study.py (hunk 1 above)
Comment (carllopez) [reply crc/models/study.py:191]: It is being set on line 184

**Pull request merged**